Desktop/ReactNativeNavigator-start/W10/Navigation.tsx

```diff
@@ -0,0 +1,83 @@
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import Ionicons from "react-native-vector-icons/Ionicons";
+import { BarCodeScannerScreen } from "./BarCodeScannerScreen";
+
+import {
+  DrawerActions,
+  NavigationContainer,
+  RouteProp,
+  StackActions,
+  useNavigation,
+  useRoute,
+} from "@react-navigation/native";
+
+import {
+  StackNavigationProp,
+  createStackNavigator,
+} from "@react-navigation/stack";
+
+import FavoritesScreen from "./FavoritesScreen";
+import { createDrawerNavigator } from "@react-navigation/drawer";
+import Profile from "./Profile";
+import AnimeDetailScreen from "./AnimeDetailScreen";
+
+const Tab = createBottomTabNavigator();
```

Comment:
These should have types associated with the navigators so that the screens know they are configured properly